Bottom: 068d3b239d365c8c8398075273f309242e460635
Top:    631bd17b7a3f48c9e75ee983fef34978345e9359
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-10 19:16:49 -0700

Refresh of delete-swig-typemaps-for-file

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
index 6cc216893c7..439c0fbe96c 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
@@ -670,7 +670,6 @@ public:
 
   static llvm::Expected<PythonFile> FromFile(File &file,
                                              const char *mode = nullptr);
-  }
 
   llvm::Expected<lldb::FileSP> ConvertToFile(bool borrowed = false);
   llvm::Expected<lldb::FileSP>
